mappings/net/minecraft/entity/mob/GhastEntity.mapping

```diff
@@ -5,7 +5,25 @@ CLASS net/minecraft/class_1916 net/minecraft/entity/mob/GhastEntity
 	METHOD method_7848 getFireballStrength ()I
 	METHOD method_7849 isShooting ()Z
 	CLASS class_1917 LookAtTargetGoal
+		FIELD field_8206 ghast Lnet/minecraft/class_1916;
+		METHOD <init> (Lnet/minecraft/class_1916;)V
+			ARG 1 ghast
 	CLASS class_1918 GhastMoveControl
+		FIELD field_8207 ghast Lnet/minecraft/class_1916;
+		FIELD field_8208 collisionCheckCooldown I
+		METHOD <init> (Lnet/minecraft/class_1916;)V
+			ARG 1 ghast
+		METHOD method_7850 willCollide (DDDD)Z
```

Comment:
`collides`

Reply:
I (and [upstream](https://github.com/FabricMC/yarn/blob/fd21c81c4ca0964220d9073018432f2ba642bfc9/mappings/net/minecraft/entity/mob/GhastEntity.mapping#L24)) think willCollide is better because the method checks if the ghast will collide with sth if it moves.

Reply:
fair enough